src/lib.rs

```diff
@@ -1408,16 +1408,38 @@ impl<'a> Face<'a> {
     /// Checks that face is marked as *Italic*.
     #[inline]
     pub fn is_italic(&self) -> bool {
+        let panose_italic = self
```

Comment:
Let's make it less functional... We can move a `panose` getter into a private function, similar to `Face::style()`.